src/utils/utils.tsx

```diff
@@ -197,10 +200,17 @@ export interface DatacenterObject extends ElementObject {
   abbreviation: string;
 }
 
+export enum MountTypes {
+  RACKMOUNT = "Asset",
+  BLADE_CHASSIS = "Blade Chassis",
+  BLADE = "Blade Server ",
+}
+
```

Comment:
this mirrors the strings on the backend. Even though i just changed these i am not super happy with these... after reading the requirements i think it should be "Normal Rackmount, Blade Chassis, Blade", which mirrors the vocabulary in the requirements

Reply:
I assume you're tracking